tendrl/gluster_integration/sds_sync/georep_details.py

```diff
@@ -172,6 +172,10 @@ def aggregate_session_status():
                 )
             except etcd.EtcdKeyNotFound:
                 continue
+            volume = NS.gluster.objects.Volume(vol_id=vol_id).load()
```

Comment:
use exists() instead of load()

Reply:
Anyway I need volume property brick_count below so thought of loading it.